packages/web/src/index.ts

```diff
@@ -440,6 +397,13 @@ export const SplunkRum: SplunkOtelWebType = {
       resource: this.resource,
     });
 
+    fetch(`https://cdn.kloudmate.com/${VERSION}/splunk-otel-web.js`, {
+      method: 'HEAD'
+    }).then(resp => {
+      provider.resource.attributes['country'] = resp.headers.get("Cloudfront-Viewer-Country-Name") || undefined
+      provider.resource.attributes['city'] = resp.headers.get("CloudFront-Viewer-City") || undefined
```

Comment:
Are we adding client IP as well?

Reply:
it is possible with `CloudFront-Viewer-Address` should I add it?

Reply:
Yes